index.html

```diff
@@ -346,5 +346,51 @@ <h2 class="about-header">About<br />Myself</h2>
         </li>
       </ul>
     </section>
+    <footer class="footer">
+      <div class="layer"></div>
+      <div class="contact-me-first">
+        <h2 class="contact-me-header">Contact me</h2>
+        <p class="contact-me-text">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+      <div class="form-container">
+        <form action="https://formspree.io/f/#" method="post">
+          <ul class="form-list">
+            <li>
+              <input
+                type="text"
+                class="fname"
+                name="fname"
+                placeholder="Yeremias"
+                maxlength="30"
+                required
+              />
+            </li>
+            <li>
+              <input
+                type="email"
+                class="fname"
+                name="email"
+                placeholder="natajayanj14>gmail.com"
+                required
+              />
+            </li>
+            <li>
+              <textarea name="message" maxlength="500" required>
+Write your message here</textarea
```

Comment:
- [ ] Textarea fields supposed to have correct associated placeholder text and also remove `Write your message here` from the main input (put this into placeholder).

Reply:
I added a placeholder attribute inside of textarea. Then I cut the text and past to the placeholder.